java-checks/src/test/files/checks/spring/SpringComponentWithWrongScopeCheck.java

```diff
@@ -0,0 +1,48 @@
+package src.test.files.checks.spring;
+import org.springframework.stereotype.Controller;
+import org.springframework.stereotype.Service;
+import org.springframework.stereotype.Repository;
+import org.springframework.context.annotation.Scope;
+import org.springframework.context.annotation.ScopedProxyMode;
+
+@Scope("prototype") // Noncompliant [[sc=1;ec=20]] {{Remove this "@Scope" annotation.}}
```

Comment:
Missing the case where a class is not annotated as well as annotated/not annotated enum/interface/annotation_type